js/gratipay/edit_team.js

```diff
@@ -0,0 +1,51 @@
+Gratipay.edit_team = {}
+
+Gratipay.edit_team.initForm = function() {
+    $form = $("#edit-team");
+    $buttons = $form.find("button");
+
+    $form.find(":input").each(function() {
+        // Save original values.
+        $(this).data('original', $(this).val());
+    });
+    $form.submit(Gratipay.edit_team.submitForm);
+}
+
+Gratipay.edit_team.submitForm = function(e) {
+    e.preventDefault();
+
+    var modified = false;
+
+    var data = new FormData();
+    $form.find(":input").each(function() {
+        if($(this).data("original") !== $(this).val()) {
+            modified = true;
+
+            var name = $(this).attr("name");
+            if (name === "image") {
+                data.append(name, $(this)[0].files[0]);
+            } else {
+                data.append(name, $(this).val());
```

Comment:
> Why would it, if it's written to do exactly that?

It is know, what happens when sometime in the future someone makes a small edit? :D This is a minor concern - the simpler our code is, the more maintainable it is, and the less bugs we'll have over time :)